collector/lib/CollectorConfig.h

```diff
@@ -161,7 +164,7 @@ class CollectorConfig {
   bool enable_processes_listening_on_ports_;
   bool import_users_;
   bool collect_connection_status_;
-  bool enable_external_ips_;
+  bool enable_external_ips_ = false;
```

Comment:
This is not needed, the value is always initialized [here](https://github.com/stackrox/collector/blob/a077c4120fe49155ad6496adbcf0d7990169d120/collector/lib/CollectorConfig.cpp#L105) and the default value is set to false [here](https://github.com/stackrox/collector/blob/a077c4120fe49155ad6496adbcf0d7990169d120/collector/lib/CollectorConfig.cpp#L52)